### DISCUSSION TITLE
In system, secure chat between users

### DISCUSSION THREAD

**Discussion by ahillbilly** [DiscipleTools/disciple-tools-theme#2868] [Ideas and Requests]
From user: 

There are certain people in the extended network that I would drop a quick message to about a situation that is related to the work but not related to a specific contact or group.  However, tracking them down, figuring out which messaging app they use (that is also secure) is often too much of a hassle.  It would be great to be able to chat with other users within the system interface.

**Comment by mikeallbutt** [comment]
So, maybe each user can assign their preferred form of receiving communication outside of the DT notifications system, and if nothing is set, DT internal gets used and/or sends emails.

**Comment by corsacca** [comment]
My guess is that the user chat will need to be a custom messaging system inside D.T as we want to keep the interface and the workflow simple. 
If we get the workflow right we could try to leverage an existing service like slack. Though D.T does try as much as possible not to depend on outside services.

@mikeallbutt, knowing a preferred way to contact a user is a good idea as a chat system is only in the idea phase. I suspect it would be hard to have cross platform messaging, both in terms of development and in terms of user adoption.
